src/partials/footer.html

```diff
@@ -26,9 +26,13 @@
 
   <!-- Link to previous and/or next page -->
   {% if page.previous_page or page.next_page %}
+    {% if page.meta and page.meta.hide %}
+      {% set hidden = "hidden" if "footer" in page.meta.hide %}
```

Comment:
perhaps we need to add a way to set this?

Reply:
Yeah, I think another meta data tag like our other `:hide-*:` mechanisms would suffice.